Dockerfile

```diff
@@ -1,31 +1,25 @@
-FROM python:3.6-jessie
+FROM ubuntu:latest
 
 LABEL maintainer="chaostoolkit <contact@chaostoolkit.org>"
 
 ADD requirements.txt requirements.txt
-RUN export DEBIAN_FRONTEND=noninteractive && \
-    apt-get update && \
-    apt-get install -y texlive-latex-base \
-      texlive-fonts-recommended \
-      texlive-latex-extra \
-      curl && \
-    curl -Lsf -o pandoc.deb https://github.com/jgm/pandoc/releases/download/2.2.1/pandoc-2.2.1-1-amd64.deb && \
-    dpkg -i pandoc.deb && \
-    rm -f pandoc.deb && \
-    pip install --no-cache-dir -U pip && \
-    pip install --no-cache-dir -r requirements.txt && \
-    pip install --no-cache-dir -U chaostoolkit chaostoolkit-reporting && \
-    apt-get remove -y curl \
-      texlive-latex-extra-doc \
-      texlive-latex-recommended-doc \
-      texlive-latex-base-doc \
-      texlive-fonts-recommended-doc \
-      texlive-pstricks-doc \
-      texlive-pictures-doc && \
-    apt-get autoremove && \
-    apt-get autoclean && \
-    rm -rf /var/lib/apt/lists/* /var/cache/apt/*
+RUN export DEBIAN_FRONTEND=noninteractive && \ 
+  apt-get update && \
+  apt-get install -y python3 \
+  python3-pip \
+  texlive-latex-base \
+  texlive-fonts-recommended \
+  texlive-latex-extra \
+  curl  \
+  pandoc  && \
+  pip install --no-cache-dir -U pip && \
+  pip install --no-cache-dir -r requirements.txt && \
+  pip install --no-cache-dir -U chaostoolkit chaostoolkit-reporting && \
+  apt autoremove && \
+  apt autoclean && \
+  rm -rf /var/lib/apt/lists/* /var/cache/apt/* 
 
+ENV MPLCONFIGDIR=/tmp/result/
```

Comment:
I'm curious why this one was added?

Reply:
This is added to address below issue while generating report:

Matplotlib created a temporary config/cache directory at /tmp/matplotlib-ip_vbjw3 because the default path (/.config/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
Usage: chaos report [OPTIONS] [JOURNAL]... REPORT
Try 'chaos report --help' for help.

Reply:
Ha I see. Makes sense.